q2_quality_control/_evaluate_taxonomy.py

```diff
@@ -0,0 +1,156 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2018, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import pandas as pd
+
+
+def _evaluate_taxonomy(exp_taxa, obs_taxa, require_exp_ids=True,
+                       require_obs_ids=True, feature_table=None,
+                       sample_id=None, level_range=range(0, 7)):
+    # validate inputs
+    join_how = _validate_indices_and_set_joining_mode(
+        exp_taxa, obs_taxa, require_exp_ids, require_obs_ids)
+
+    # merge tables
+    taxa = exp_taxa.join(
```

Comment:
In my hands-on testing I get the following error:

```bash
~/data/moving-pictures
☻  qiime quality-control evaluate-taxonomy \
--i-expected-taxa taxonomy.qza \
--i-observed-taxa taxonomy.qza \
--p-depth 7 \
--o-visualization eval-taxa.qzv \
else> --verbose
Traceback (most recent call last):
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/q2cli/commands.py", line 274, in __call__
    results = action(**arguments)
  File "<decorator-gen-383>", line 2, in evaluate_taxonomy
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/qiime2/sdk/action.py", line 231, in bound_callable
    output_types, provenance)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/qiime2/sdk/action.py", line 428, in _callable_executor_
    ret_val = self._callable(output_dir=temp_dir, **view_args)
  File "/Users/matthew/src/qiime2/q2-quality-control/q2_quality_control/quality_control.py", line 104, in evaluate_taxonomy
    level_range=range(0, depth))
  File "/Users/matthew/src/qiime2/q2-quality-control/q2_quality_control/_evaluate_taxonomy.py", line 20, in _evaluate_taxonomy
    taxa = exp_taxa.join(
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/pandas/core/generic.py", line 3614, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'Series' object has no attribute 'join'

Plugin error from quality-control:

  'Series' object has no attribute 'join'

See above for debug info.
```

Running pandas 0.23.1.

Tests pass locally, looks like you are feeding in dataframes in the tests, but the method uses series'.

Reply:
yikes... not sure how that slipped through, since I tested with QZAs locally (this was written in the pandas 0.22 days but I doubt this was a change they made since then)

have changed the method to view as `DataFrame` instead... now tests and hands-on testing both work here. Pls let me me know if your hands-on tests now pass.